cost/superseded_instance/superseded_instance.pt

```diff
@@ -0,0 +1,412 @@
+  run_script $js_format_costs, $ds_new_bc_costs, $ds_cloud_vendor_accounts, $ds_billing_centers
+end
+
+script "js_format_costs", type: "javascript" do
+  parameters "new_bc_costs", "ds_cloud_vendor_accounts", "ds_billing_centers"
+  result "formatted_data"
+  code <<-EOS
+  //https://www.w3resource.com/javascript-exercises/javascript-date-exercise-3.php
+  var getDaysInMonth = function(month,year) {
+    // Here January is 1 based
+    //Day 0 is the last day in the previous month
+    return new Date(year, month, 0).getDate();
+  // Here January is 0 based
+  // return new Date(year, month+1, 0).getDate();
+  };
+  var date = new Date()
+  var today = date.getDate()
+  var numdays = getDaysInMonth(date.getUTCMonth(),date.getUTCFullYear())
+  var monthcomplete = today / numdays ;
+
+  var formatted_data = [];
+  var unsorted_results = [];
+  var arr_results = [];
+  var bcs = [];
+  _.each(new_bc_costs, function(bcc){bcs.push(bcc.id)})
+  bcs = _.uniq(bcs) ;
+  _.each(bcs, function(bc_id){
+    var new_bc_cost_objs = _.reject(new_bc_costs, function(new_bc_cost){ return new_bc_cost.id != bc_id });
+    _.each(new_bc_cost_objs, function(new_bc_cost_obj){
+      var billing_center = _.reject(ds_billing_centers, function(bc){ return bc.id != new_bc_cost_obj.id })[0];
+      if (new_bc_cost_obj.vendor == "AWS" ) {
+        var account = _.reject(ds_cloud_vendor_accounts, function(ds_cloud_vendor_account){ return ds_cloud_vendor_account.id != new_bc_cost_obj.vendor_account_name });
+        var vendor_account_name = account[0].name
+      } else {
+        var vendor_account_name = new_bc_cost_obj.vendor_account_name
+      }
+      cost = new_bc_cost_obj.cost_amortized_unblended_adj
+      run_rate = cost / monthcomplete
+      if (new_bc_cost_obj.cost_amortized_unblended_adj != 0){
+        arr_results.push({
+          name: billing_center.name,
+          id: bc_id,
+          new_sum: new_bc_cost_obj.cost_amortized_unblended_adj,
+          service: new_bc_cost_obj.service,
+          vendor: new_bc_cost_obj.vendor,
+          vendor_account_name: vendor_account_name,
+          usage_amount: new_bc_cost_obj.usage_amount,
+          instance_type: new_bc_cost_obj.instance_type,
+          resource_id: new_bc_cost_obj.resource_id,
+          resource_type: new_bc_cost_obj.resource_type,
+          timestamp: new_bc_cost_obj.timestamp,
+          run_rate: parseFloat(run_rate).toFixed(2),
+          run_rate_for_calc: run_rate
+        })
+      }
+    })
+  })
+  var total =  parseFloat(_.reduce(_.pluck(arr_results, "run_rate_for_calc"), function(memo, num){ return memo + num; }, 0)).toFixed(2);
+  _.each(arr_results, function(result){ 
+    result["total"] = total
+    formatted_data.push(result)
+  })
+EOS
+end
+
+datasource "ds_aws_instance_size_map" do
+  request do
+    verb "GET"
+    host "raw.githubusercontent.com"
+    path "/rightscale/policy_templates/aws_superseded_instances/data/aws/instance_types.json"
```

Comment:
I don't think this path is right, and should this use the new data path.  The new path is mentioned in the README.